docs/AI-for-security/ai-security-assistant.asciidoc

```diff
@@ -47,7 +47,7 @@ You must create a generative AI connector before you can use AI Assistant. AI As
 .Recommended models
 [sidebar]
 --
-While AI Assistant is compatible with many different models, our testing found increased quality with Azure 32k, and faster, more cost-effective responses with Claude 3 Haiku and OpenAI GPT4 Turbo. For more information, refer to the <<llm-performance-matrix>>.
+While AI Assistant is compatible with many different models, we recommend you refer to the <<llm-performance-matrix>> to select models that perform well with your desired use cases.
```

Comment:
Feels unnecessary, since this isn't a specific recommendation, just a reference to another docs page:
```suggestion
While AI Assistant is compatible with many different models, refer to the <<llm-performance-matrix>> to select models that perform well with your desired use cases.
```